External/Plugins/ASCompletion/Completion/ASGenerator.cs

```diff
@@ -2793,22 +2792,10 @@ public static void GenerateExtractVariable(ScintillaControl Sci, string NewName)
                         }
                     }
 
-                    int style = Sci.StyleAt(funcBodyStart + lastPos) & stylemask;
-                    if (ASComplete.IsCommentStyle(style))
```

Comment:
The only condition is that we are interested in right now checking for comments.